src/main/java/org/folio/service/pieces/PieceStorageService.java

```diff
@@ -196,7 +196,9 @@ static String getQueryForUserTenants(List<String> userTenants, String query) {
     String tenantCql = convertIdsToCqlQuery(userTenants, "receivingTenantId");
     String nullTenantCql = getCqlExpressionForFieldNullValue("receivingTenantId");
     String combinedTenantCql = combineCqlExpressions("or", tenantCql, nullTenantCql);
-    return combineCqlExpressions("and", query, combinedTenantCql);
+    return query != null
```

Comment:
small suggestion: possible case, maybe like StringUtils.isNotBlank in case of query is " " or query =""